com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/Logic/NandNode.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class NandNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,     // Version
+            "Nand", // Name
+            "Out = !A && !B;",
+            new ParameterDescriptor("A", TYPE.Bool, Usage.In),
+            new ParameterDescriptor("B", TYPE.Bool, Usage.In),
```

Comment:
same question as the And Node, do we want the input ports to also accept vectors?

Reply:
They will accept and auto-cast other data types.